services/EnsService.ts

```diff
@@ -95,6 +95,9 @@ export default class EnsService {
           resolver.getText(textKey).then((result) => {
             let proceedWithSettingRecord = true;
             if(textKey === APP_BSKY) {
+              if(result === null || result === '') {
```

Comment:
@makhnovskiy try to `trim()` the variable prior comparing to an empty string, otherwise `    ` would pass this condition